src/ophyd_async/panda/writers/_panda_hdf_file.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
```

Comment:
As above RE: multiple implementations of HDFFile

Reply:
Cleaned up via https://github.com/bluesky/ophyd-async/pull/245/commits/a26d6245a2e8b55c0a53435d18a8aa5bc8fd193d.